.github/workflows/e2e-init-chain.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and Push E2E Init Chain Images
+
+on:
+  push:
+    tags:
+    - 'v12+.[0-20]+.0+' 
+    - 'v[0-20]+.0+.0+' 
```

Comment:
Why not this?

https://github.com/osmosis-labs/osmosis/blob/8ff4c7fe52657319f6b3c299c7002f7aa90d2d7b/.github/workflows/docker.yml#L27

Reply:
In the issue, you suggested : We should auto-publish the above image on any minor (12.X.0) and major (X.0.0) release. So I wrote it like that